doc/reference/causality-overview.rst

```diff
@@ -0,0 +1,24 @@
+Overview
+--------
+This module provides function to estimate causal influences of signals on each other.
+
+Granger causality
+~~~~~~~~~~~~~~~
+Granger causality is a method to determine causal influence of one signal on another based on autoregressive modelling. It was developed by Nobel prize laureate Clive Granger and has been adopted in various numerical fields ever since [1]. In its simplest form, the method tests whether the past values of one signal help to reduce the prediction error of another signal, compared to the past values of the latter signal alone. If it does reduce the prediction error, the first signal is said to Granger cause the other signal.
+
+
+Limitations
+"""""""""""
+The user must be mindful of the method's limitations, which are assumptions of covariance stationary data, linearity imposed by the underlying autoregressive modelling as well as the fact that the variables not included in the model will not be accounted for [2].
+
+References
+""""""""""
+[1] Granger CWJ (1969) Investigating causal relations by econometric models and cross-spectral methods. Econometrica 37:424-438.
+
+[2] Seth A (2007) Granger causality. Scholarpedia, 2(7):1667., revision #127333
```

Comment:
We start using bibtex for references. So you need to move the references to `doc/bib/elephant.bib` and then link them here. For example, see how it's done in unitary events analyses and asset. It should be something like

Add in `doc/bib/elephant.bib`:

`@article{yourcustomcausalityreferencename, ...`

Add in `doc/reference/causality`:

```

References
----------

.. bibliography:: ../bib/elephant.bib
   :labelprefix: gr (or ca)
   :keyprefix: granger- (or causality-)
   :style: unsrt
```

In `.rst` or `.py` file, where the documentation is described (in your case, it's `causality-overview`), use it as 
":cite:\`granger-yourcustomcausalityreferencename\`" in-place.

Reply:
This has been done in commits 632604a and e15b99b.

Reply:
Please note that I have an arxiv preprint in the bibliography. This text was later published as a book chapter.

Reply:
> Please note that I have an arxiv preprint in the bibliography. This text was later published as a book chapter.